### PR TITLE
Adding an animation action that can spawn projectiles 

### PR DESCRIPTION
**Pull request by Interrupt**
projectile-attack-animation-2

### CODE REVIEW

File: Dungeoneer/src/com/interrupt/dungeoneer/entities/Monster.java [hunk 1]
```diff
@@ -1175,6 +1130,90 @@ public void attack(Entity target)
 		}
 	}
 
+	private void rangedAttack(Entity target) {
+		if(target == null || !target.isActive) return;
+		if(!hostile) return;
+		if(stuntime > 0) return;
+		if(isParalyzed()) return;
+		if(!alerted) alerted = true;
```
Comment (joshuaskelly) [inline Dungeoneer/src/com/interrupt/dungeoneer/entities/Monster.java:1138]: Could be simplified to:
`alerted = true;`

**Review by joshuaskelly — APPROVED** [review]
One little nit, otherwise great work! 🏆

**Comment by joshuaskelly** [comment]
Question: will potions behave the same as if the player threw them?

**Comment by Interrupt** [comment]
@joshuaskelly now they will!

**Pull request merged**